src/allotropy/parsers/chemometec_nucleoview/nucleoview_reader.py

```diff
@@ -0,0 +1,102 @@
+import io
+
+import numpy as np
+import pandas as pd
+
+
+def convert_datetime(x: pd.Series) -> pd.Series:
+    return pd.to_datetime(x).dt.strftime("%Y-%m-%d %H:%M:%S%z")
+
+
+def convert_float(x: pd.Series) -> pd.Series:
+    return pd.to_numeric(x, errors="coerce")
+
+
+def convert_int(x: pd.Series) -> pd.Series:
+    return pd.to_numeric(x, errors="coerce")
+
+
+def convert_string(x: pd.Series) -> pd.Series:
```

Comment:
Unless you intend these functions to be called outside of this package, could you please "protect" them? E.g., `_convert_string`.

Reply:
removed from latest refactor